contributing.md

```diff
@@ -0,0 +1,141 @@
+# Contributing
+
+> You want to contribute to the website? Welcome!
+> You will find here resources and advices that could help you.
+
+If you have any questions, you can ask them in #website-dev channel in our Discord.
+
+ * [Preparing the development environment](#preparing-the-development-environment)
+ * [Guidelines](#guidelines-1)
+ * [Localization](#localization)
+
+<!-- /code_chunk_output -->
+
+
+## Preparing the development environment
+### Fork the project
+* Fork the repo.
+* **Always create a new branch before working on your fork.**
+* You can now improve the website.
+
+> You can use a git client like [GitHub Desktop](https://desktop.github.com/) to manage your local repository.
+
+When your done submit a pull request.
+
+### Running the website
+> When you make modifications to the site it can be useful to render the code. You will thus see how the modifications look like.
+
+See [Building local enviroment](https://github.com/Paperback-iOS/website#building-local-enviroment)
+
+## Guidelines
+### Glossary
+
+ * App/application --> app
+ * Device/iPhone/iPad/phone --> iPhone or iPad
+ * iOS --> iOS or iPadOS
+
+> Be careful of capitalization:
+
+* Paperback
+* MangaDex
+
+* AltStore
+* AltServer
+
+* Safari
+* macOS
+
+### Syntax and layout
+> To create a numbered list use `1.` on all lines:
+```markdown
+1. Line 1
+1. Line 2
+```
+
+> Write `.IPA` in a code block.
```

Comment:
Clarify a little more about this! Maybe give an example of what you mean exactly? Instructions are a little vague.

Reply:
I removed this line. People should not need to use `.IPA` often and if they use .IPA, we will tell them to change it to `.IPA`